### ISSUE TITLE
obs crashing when selecting (update values from amf)

### ISSUE THREAD

**Issue by wazerstar** [Xaymar/OBS-AMD-Advanced-Media-Framework#44]
Hardware:
CPU: Intel 4790K
Graphics: AMD R9 290 4GB GDDR5 modded to 290x - Then remodded to 390x
Motherboard: Asus MAXIMUS VI FORMULA
Ram: 16GB DDR3 2400mhz
Disk: 1TB Samsung EVO 840

Logs
[logs.zip](https://github.com/Xaymar/OBS-AMD-Media-Framework/files/433118/logs.zip)

**Comment by Xaymar** [comment]
Same issue as before with AMF_ACCESS_DENIED, normally it should've been caught by try...catch.

```
19:11:24.102: [AMF Encoder] <Plugin::AMD::H264VideoEncoder::GetNumberOfTemporalEnhancementLayers> Retrieving Property failed with error AMF_ACCESS_DENIED (code 3).
```

**Comment by wazerstar** [comment]
Except with the old 1.2x it still showed the same error but did not crash.

**Comment by Xaymar** [comment]
Can't reproduce this here with 1.3.0pre3 or 1.2.0rc1.

**Comment by Xaymar** [comment]
Seems to be fixed in 1.3.0pre3/pre4, doesn't happen in Release or Debug builds.
